test/cases/gcp.sh

```diff
@@ -287,6 +287,9 @@ fi
 
 verifyInGCP
 
+# Run cleanup function
+cleanup
```

Comment:
If `verifyInGCP` fails, the `cleanup` still won't run, right?

Reply:
Yeah, then it's up to the scheduled cloud cleaner... Maybe I could trap the cleanup again after the line where the traps are removed instead?

Reply:
Yep, that makes sense.